### ISSUE TITLE
Issue with gmlas.d.10: Validate geometries (1) regarding outer ring orientation

### ISSUE THREAD

**Issue by heidivanparys** [inspire-eu-validation/ets-repository#137]
There is an issue with gmlas.d.10: Validate geometries (1), as also reported in #60 .

When providing a GML file in http://www.opengis.net/def/crs/EPSG/0/4326, with coordinates transformed from http://www.opengis.net/def/crs/EPSG/0/25832, the validation fails with message "Error detected: Invalid polygon. Outer ring of polygon is clockwise within element MultiSurface". Transformation was done by two different tools (Go Publisher and FME), and the validation fails for both resulting files.

When providing a modified GML file, where the points of the exterior boundary are placed in reverse order, the validation passes.

When validating the same files with Test Suite: GML 3.2 (ISO 19136:2007) Conformance Test Suite from http://cite.opengeospatial.org/teamengine/, the results are opposite. That is, the original file, created by the tools, passes validation, the modified file fails.

So to these two validators give a conflicting result.

When visualising the polygon with the coordinates from EPSG:4326, the orientation of the boundary is counterclockwise, thus its orientation is positive, thus it should not fail the test, as I see it.

When validating with GML file in http://www.opengis.net/def/crs/EPSG/0/25832, there is no issue. So this may have something to do with the axis order of the coordinate system used.

[Test which reports incorrect orientation for EPSG:4326](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EIDec18c8bb-3fec-4d18-b79e-3578cba97028.html?lang=en#EIDeeb34caa-ca3e-46ac-8170-a494d032b7b3), with test object
[GPP_ProtectedSites_a113a810_EPSG4326.txt](https://github.com/inspire-eu-validation/ets-repository/files/1646763/GPP_ProtectedSites_a113a810_EPSG4326.txt)

Result in OGC validator:
![image](https://user-images.githubusercontent.com/11427611/35156124-1f9899ec-fd30-11e7-810b-0c0bd2c093b0.png)

[Test which reports correct orientation for EPSG:4326](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EIDa0561f64-a035-4a40-bd26-76803785d198.html?lang=en#EIDf916edd9-2b62-4a57-a54f-04eaafe5d9b4), with test object 
[GPP_ProtectedSites_a113a810_EPSG4326_inverseorientation.txt](https://github.com/inspire-eu-validation/ets-repository/files/1646765/GPP_ProtectedSites_a113a810_EPSG4326_inverseorientation.txt)

Result in OGC validator:
![image](https://user-images.githubusercontent.com/11427611/35156096-0efdb086-fd30-11e7-82f2-b3c5767382b2.png)

[Test with data in EPSG:4326](http://inspire-sandbox.jrc.ec.europa.eu/etf-webapp//v2/TestRuns/EIDe845e7fd-03f8-4c45-bbf9-fbb4b1aba5a2.html?lang=en#EIDdd894d18-2356-4f9a-b962-e31058b9db67), with test object [ProtectedSites_a113a810-53a7-11e2-b325-00155d01e765_EPSG25832.txt](https://github.com/inspire-eu-validation/ets-repository/files/1646811/ProtectedSites_a113a810-53a7-11e2-b325-00155d01e765_EPSG25832.txt)

![image](https://user-images.githubusercontent.com/11427611/35155846-40ef5938-fd2f-11e7-8d98-5c796e3babc6.png)

**Comment by cportele** [comment]
I agree, this is not satisfactory. And different libraries have implemented different interpretations. See the discussion in #60.

I will have a look in the CITE test to see how they determine the orientation...

**Comment by cportele** [comment]
Yes, the [CITE tests](https://github.com/opengeospatial/ets-gml32/blob/master/src/main/java/org/opengis/cite/iso19136/data/spatial/GeometryAssert.java) convert the geometry to a right-handed CRS before determining the orientation - also using JTS.  A comment states that JTS would expect a right-handed CRS, but I found nothing in the JTS documentation or source code about this. I guess it boils down to the question whether the orientation of a geometry depends on the left/right-hand nature of the CRS.

**Comment by carlospzurita** [comment]
We are moving forward with this issue. The pull request https://github.com/deegree/deegree3/pull/1034 with a fix in deegree to be used in the ETF code has been accepted. Whenever the new release is ready, we will integrate it on the validator. Will keep you posted.

**Comment by fabiovinci** [comment]
Dear all,

a fix for this issue is available in the [BETA instance](http://beta-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/home/index.html) of the validator, could you please test the solution and report your feedback?

For reporting issues, please use the [Validator helpdesk](https://github.com/INSPIRE-MIF/helpdesk-validator/issues).

**Comment by heidivanparys** [comment]
It works fine now in the beta instance, thank you.
